packages/rpc-graphql/src/schema/scalars.ts

```diff
@@ -0,0 +1,20 @@
+import { GraphQLScalarType, Kind } from 'graphql';
+
+export const BigIntScalar = new GraphQLScalarType({
+    name: 'BigInt',
+
+    parseLiteral(ast) {
+        if (ast.kind === Kind.STRING) {
+            return BigInt(ast.value);
+        }
+        return null;
+    },
+
+    parseValue(value) {
+        return BigInt(value as string | number | bigint | boolean);
+    },
```

Comment:
How come this isn't just `value as string` if you're storing the value as a string? Or I guess are there cases where `value` in this function would not be a string?